pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -138,13 +124,15 @@ buildPythonPackage rec {
     find "$IBIS_TEST_DATA_DIRECTORY" -type f -exec chmod u+rw {} +
 
     # load data
-    for backend in ${backendsString}; do
-      python ci/datamgr.py "$backend" &
+    for backend in ${lib.concatStringsSep " " backends}; do
+      python ci/datamgr.py load "$backend"
```

Comment:
Please use `${python.interpreter}` instead of python.

Reply:
Fixed.